From a8b77ea8444b6186c250449655dbe940e5a1089e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 18 Sep 2018 05:12:42 +0000
Subject: [PATCH 1/3] push doc: add spacing between two words
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix a formatting error introduced in my recently landed
fe802bd21e ("push doc: correct lies about how push refspecs work",
2018-08-31).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/git-push.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/git-push.txt b/Documentation/git-push.txt
index f345bd30fc..a5fc54aeab 100644
--- a/Documentation/git-push.txt
+++ b/Documentation/git-push.txt
@@ -110,7 +110,7 @@ can be overridden by adding an the optional leading `+` to a refspec
 is that no amount of forcing will make the `refs/heads/*` namespace
 accept a non-commit object. Hooks and configuration can also override
 or amend these rules, see e.g. `receive.denyNonFastForwards` in
-linkgit:git-config[1] and`pre-receive` and `update` in
+linkgit:git-config[1] and `pre-receive` and `update` in
 linkgit:githooks[5].
 +
 Pushing an empty <src> allows you to delete the <dst> ref from the
-- 
2.19.0.rc2.392.g5ba43deb5a

